frontend/src/Page/EditBikePage/EditBikePage.tsx

```diff
@@ -0,0 +1,26 @@
+import {Bike} from "../../model/Bike";
+import useAuth from "../../Hooks/useAuth";
+import {useParams} from "react-router-dom";
+import ResponsiveAppBar from "../../Component/ResponsiveAppBar/ResponsiveAppBar";
+import EditBikeForm from "../../Component/EditBikeForm/EditBikeForm";
+import {Typography} from "@mui/material";
+
+type EditBikePageProps = {
+    bikes: Bike[]
+    updateBikeList(bikes: Bike[]): void
+}
+export default function EditBikePage(props: EditBikePageProps){
+    useAuth(true)
+    const { bikeId } = useParams<{ bikeId: string }>()
+    const bike: Bike | undefined = props.bikes.find((bike) => bike.id === bikeId);
```

Comment:
What happens if the `find` doesn't return a bike? I would add a conditional render for that case :)